packages/client/src/hooks/useContacts.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useReducer, useMemo } from "react";
+import reducer, { INITIAL_STATE } from "../reducers/contacts";
+
+const storageKey = "vessel-contacts";

```

Comment:
Can we move this constant to the constants/constants.js file?

Reply:
I would prefer to have this co-located with the reducer it belongs to, similar to how our other storage keys are set. It's only used once per hook.

Reply:
From reusability perspective it does make sense to keep it close to the data, but I would argue that since this is a very high level configuration (equivalent to a db config) sprinkling them across the project makes it hard to edit and understand. Looking into the future, we probably will move into using different strategy for data storage (indexDB or a decentralized db), having configurations in one single file will help switching the strategy